ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -114,16 +114,16 @@
 public class AcidUtils {
   // This key will be put in the conf file when planning an acid operation
   public static final String CONF_ACID_KEY = "hive.doing.acid";
-  public static final String BASE_PREFIX = "base_";
+  public static final String BASE_PREFIX = AcidConstants.BASE_PREFIX;
```

Comment:
could we remove those and reference constants from AcidConstants directly

Reply:
I am not sure I want to remove these public constants, the AcidUtils is unfortunately is being used by spark-acid and it might break compatibility...